app/src/main/res/values/styles.xml

```diff
@@ -2,75 +2,197 @@
 
     <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
         <!--COLORS-->
-        <item name="colorPrimary">@color/sanMarino</item>
-        <item name="colorPrimaryDark">@color/resolutionBlue</item>
+        <item name="colorPrimary">@color/custom_blue</item>
+        <item name="colorPrimaryDark">@color/custom_blue_dark</item>
         <item name="colorOnPrimary">@color/white</item>
-        <item name="colorAccent">@color/wildStrawberry</item>
-        <!--TEXT-->
-        <item name="textAppearanceHeadline1">@style/TextAppearance.RealEstateManagerStyles.Headline1</item>
-        <item name="textAppearanceHeadline3">@style/TextAppearance.RealEstateManagerStyles.Headline3</item>
-        <item name="textAppearanceHeadline4">@style/TextAppearance.RealEstateManagerStyles.Headline4</item>
-        <item name="textAppearanceHeadline5">@style/TextAppearance.RealEstateManagerStyles.Headline5</item>
-        <item name="textAppearanceSubtitle1">@style/TextAppearance.RealEstateManagerStyles.Subtitle1</item>
-        <item name="textAppearanceBody1">@style/TextAppearance.RealEstateManagerStyles.Body1</item>
+        <item name="colorAccent">@color/custom_pink</item>
     </style>
 
-
     <!--__________________________________COMMON STYLES__________________________________-->
 
-    <style name="Widget.TextViewTitleStyle" parent="Widget.MaterialComponents.TextView">
+    <style name="Widget.Headline1TextView" parent="Widget.MaterialComponents.TextView">
         <item name="android:paddingBottom">@dimen/basic_marge</item>
-        <item name="android:textAppearance">@style/TextAppearance.RealEstateManagerStyles.Headline4</item>
+        <item name="android:layout_marginStart">@dimen/basic_marge</item>
+        <item name="android:textAppearance">
+            @style/TextAppearance.RealEstateManagerStyles.Headline1
+        </item>
+    </style>
+
+    <style name="Widget.Subtitle1TextView" parent="Widget.MaterialComponents.TextView">
+        <item name="android:textAppearance">
+            @style/TextAppearance.RealEstateManagerStyles.Subtitle1
+        </item>
+        <item name="paddingStart">@dimen/little_marge</item>
     </style>
 
     <!--__________________________________CUSTOM VIEW__________________________________-->
 
-    <style name="customViewInformationIconStyle" >
+    <style name="customViewIconStyle">
         <item name="android:paddingEnd">@dimen/basic_marge</item>
-        <item name="android:paddingStart">0dp</item>
-        <item name="srcCompat">@drawable/ic_fragment_detail_area_24</item>
     </style>
 
-    <!--TITLE-->
```

Comment:
Same here. TextAppearence is not a widget.

Reply:
done